src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -543,17 +549,28 @@ public void ClearSearchTextBox(object obj)
         /// one of these toast notifications.
         /// </summary>
         /// <param name="obj"></param>
-        public void ClearDownloadToastNotification(object obj)
+        public void ClearToastNotification(object obj)
```

Comment:
This is API breaking but I guess you decided to break it intentionally?

Reply:
Yes to make this function a bit more generic